cmd/tk/init.go

```diff
@@ -22,8 +25,7 @@ func initCmd() *cli.Command {
 	}
 
 	force := cmd.Flags().BoolP("force", "f", false, "ignore the working directory not being empty")
-	installK8sLibFlag := cmd.Flags().Bool("k8s", true, "set to false to skip installation of k8s-alpha")
-	installK8sVersion := cmd.Flags().String("k8sversion", "1.20", "choose the version of k8s-alpha")
+	installK8s := cmd.Flags().String("k8s", defaultK8sVersion, "choose the version of k8s-alpha, set to false to skip")
```

Comment:
no shorthand version?

Reply:
Would that be better? This approach keeps compatibility with `--k8s=<bool>` flag.